### PR TITLE
[SIMPLE_FORMS] feat: update form 21-10210 with new pdf form and expiration date

### PR DESCRIPTION
**Pull request by pennja**
## Summary

- This work updates the form PDF template, form mappings, and stamp locations for newly updated form 21-10210

## Related issue(s)

- [Update form 21-10210 with new PDF form and Expiration date](https://app.zenhub.com/workspaces/vagov-product-team-forms-634853151f5c6000165942bc/issues/gh/department-of-veterans-affairs/va.gov-team-forms/1571)

## Testing done

- [x] Logic is already covered by unit tests

## Requested Feedback

Any

### CODE REVIEW

File: modules/simple_forms_api/app/models/simple_forms_api/vba_21_10210.rb [hunk 1]
```diff
@@ -34,19 +34,19 @@ def words_to_remove
     end
 
     def desired_stamps
-      [{ coords: [50, 160], text: data['statement_of_truth_signature'], page: 2 }]
+      [{ coords: [50, 195], text: data['statement_of_truth_signature'], page: 2 }]
```
Comment (Thrillberg) [inline modules/simple_forms_api/app/models/simple_forms_api/vba_21_10210.rb:37]: Good thinking, updating these too! 👍

**Review by Thrillberg — APPROVED** [review]
Love the comments in the `erb`! Nice work!

File: modules/simple_forms_api/spec/services/pdf_filler_spec.rb [hunk 2]
```diff
@@ -76,8 +89,12 @@
     end
 
     def read_form_mapping(form_number)
```
Comment (pennja) [inline modules/simple_forms_api/spec/services/pdf_filler_spec.rb:91]: I'm not convinced this test was ever properly populating the values for this JSON string. It definitely does now.

Comment (Thrillberg) [reply modules/simple_forms_api/spec/services/pdf_filler_spec.rb:91]: ooo good catch!

**Review by Thrillberg — APPROVED** [review]
Looks great! Thanks for the spec fix too!

**Review by stevenjcumming — APPROVED** [review]

**Pull request merged**